File: fuckApp/src/main/java/com/fuckapp/fragment/view/AppView.java
package com.fuckapp.fragment.view;

import com.fuckapp.fragment.model.AppModel;

import java.util.List;

/**
 * by y on 2016/10/31
 */

public interface AppView {
    void removeAllAdapter();

    void setAppInfo(List<AppModel> appInfo);

    void showProgress();

    void hideProgress();

    void obtainSuccess();

    void obtainError();
}

File: FuckApp/app/src/main/java/com/fuckapp/fragment/view/AppView.kt
package com.fuckapp.fragment.view

import com.fuckapp.fragment.model.AppModel

/**
 * by y on 2016/10/31
 */

interface AppView {
    fun removeAllAdapter()

    fun setAppInfo(appInfo: List<AppModel>)

    fun showProgress()

    fun hideProgress()

    fun obtainSuccess()

    fun obtainError()
}
